lib/internal/crypto/keys.js

```diff
@@ -639,8 +643,8 @@ function createPrivateKey(key) {
   return new PrivateKeyObject(handle);
 }
 
-function isKeyObject(key) {
-  return key instanceof KeyObject;
+function isKeyObject(obj) {
+  return obj != null && obj[kHandle] !== undefined;
```

Comment:
I realize now `kHandle` is used all over the place. @tniessen, is there a symbol unique to KeyObject we can use instead?